src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -72,8 +72,8 @@
     /** Optional ID attribute of a test suite. E.g., Eclipse plug-ins tests always have the name 'tests' but a different id. **/
     private String id;
 
-    /** Optional time attribute of a test suite. E.g., Suites can use their own time attribute or the sum of their cases' times as before.**/
-    private String time;
```

Comment:
If the class is being persisted on the disk, it requires a migration logic in readResolve()